tests/e2e/template_cleantest_test.go

```diff
@@ -83,39 +83,52 @@ var _ = Describe("Example of a clean test", func() {
 
 		})
 
-		var _ = Context("when --context is used", func() {
-			var _ = Context("when project from KUBECONFIG is used", func() {
-				// Set active project for each test spec
-				var _ = JustBeforeEach(func() {
-					helper.OcSwitchProject(project)
-				})
-				// go back to original project after each test
-				var _ = JustAfterEach(func() {
-					helper.OcSwitchProject(originalProject)
-				})
-
-				It("create local nodejs component and push code", func() {
-					helper.CopyExample(filepath.Join("source", "nodejs"), context)
-
-					helper.CmdShouldPass("odo component create nodejs nodejs --context " + context)
-					//TODO: verify that config was properly created
-					helper.CmdShouldPass("odo push")
-					//TODO: verify resources on cluster
-				})
+	})
+
+	var _ = Context("when --context is used", func() {
+
+		// we will be testing components that are created from the current directory
+		// switch to the clean context dir before each test
+		var _ = JustBeforeEach(func() {
```

Comment:
This shouldn't be here. If you are calling commands with `--context` flag you shouldn't to the directory, otherwise, there is no point of specifying `--context`

Reply:
ack, I felt so too.

Reply:
done